src/server/server_family.cc

```diff
@@ -1071,7 +1071,9 @@ void PrintPrometheusMetrics(const Metrics& m, DflyCmd* dfly_cmd, StringResponse*
                             MetricType::GAUGE, &resp->body());
   AppendMetricWithoutLabels("blocked_clients", "", conn_stats.num_blocked_clients,
                             MetricType::GAUGE, &resp->body());
-  AppendMetricWithoutLabels("pipeline_queue_bytes", "", conn_stats.dispatch_queue_bytes,
```

Comment:
wow

Reply:
that's something I added/renamed a few days ago @chakaz . Why it's not good?

Reply:
We have 2 terms - `dispatch_queue_bytes` and `pipeline_cache_bytes`, which we use both in the code (variable names etc) and in `INFO`:

https://github.com/dragonflydb/dragonfly/pull/3117/files#diff-04447876005188a028fe0e46ee54f437b03f9d2dfb8b456be2fa6fa539a56499R2032-R2033 

The name `pipeline_queue_bytes` is confusing because it "sounds" like it's `pipeline_cache_bytes`, but in fact it has the value of `dispatch_queue_bytes` :)